api/envoy/protobuf/descriptor.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package envoy.protobuf;
+
+option java_outer_classname = "DescriptorProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.protobuf";
+
+import "google/protobuf/descriptor.proto";
+
+extend google.protobuf.FieldOptions {
+  // This option indicates that a field contains personally-identifying or otherwise sensitive data,
+  // such as a private key or a password. It is used by `MessageUtil::redact` to determine which
+  // fields need to be sanitized. Please note that this has no effect on standard Protobuf functions
+  // such as `TextFormat::PrintToString`; you must explicitly call `MessageUtil::redact` wherever
```

Comment:
Can we add `check_format` support to ban all the likely candidates (e.g. `DebugString`, `PrintToString`) and force safe uses?

Reply:
Let's chat about that tomorrow; I'm not sure how big a hammer we want to use, if any.